service-api/module/Application/src/Controller/IdentityController.php

```diff
@@ -63,7 +63,16 @@ public function createAction(): JsonModel
                 'address'       => ['SS' => $caseData->toArray()['address']]
             ];
 
-            $this->dataImportHandler->insertData($item);
+            $insert = $this->dataImportHandler->insertData($item);
+
+            if (! $insert) {
+                $this->getResponse()->setStatusCode(Response::STATUS_CODE_422);
+                return new JsonModel(new Problem(
+                    'Data save failed / invalid data format',
+                    extra: ['errors' => 'Please see logs'],
```

Comment:
We don't need to tell the front end to check the logs, this is implicit in our processes
```suggestion
```